CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,81 @@
+
+## What is not okay?
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or advances of any kind.
+* Trolling, insulting or derogatory comments, and personal or political attacks.
+* Public or private harassment.
+* Publishing others’ private information, such as a physical or email address, without their explicit permission.
+* Other conduct which could reasonably be considered inappropriate in a professional setting.
+
+
+## Where does this Code of Conduct apply?
+
+This code of conduct covers anyone participating in PureScript community channels, including our Discord server, Discourse forum, and GitHub properties. For members of the PureScript team, it applies anywhere they are representing the PureScript project (for example, on social media).
+
+## How do members report violations?
+
+You can report abusive, harassing, or otherwise unacceptable behavior to the community moderators team or to a particular moderator if you are not comfortable sharing your concern with the full team.
+
+All complaints will be reviewed and investigated promptly and fairly. Your communication with moderators will not be shared with anyone else without your permission.
+
+Here are per-platform instructions:
+
+* Discord: Send a private message to the `@Moderators` team. To find a specific moderator to message, use the members sidebar under the `Moderators` role to find their profile.
+* Discourse: Send a private message to the `moderators` group. To find a specific moderator to message, see the `moderators` group page: https://discourse.purescript.org/g/moderators
+* All Other: If an event occurs somewhere else, please reach out to moderators on either Discord or Discourse.
+
+## How will you enforce the rules?
+
+Our community leaders follow the below guidelines to determine the consequences for any action they deem in violation of this Code of Conduct:
+
+1. **Correction**
+
+   Cause: Use of inappropriate language or other behavior deemed unprofessional or unwelcome in the community.
+
+   Consequence: A private, written warning from community leaders, providing clarity around the nature of the violation and an explanation of why the behavior was inappropriate. A public apology may be requested.
+
+2. **Warning**
+
+   Cause: A violation through a single incident or series of actions.
+
+   Consequence: A warning with consequences for continued behavior. No interaction with the people involved, including unsolicited interaction with those enforcing the Code of Conduct, for a specified period of time. Violating these terms may lead to a temporary or permanent ban.
+
+3. **Temporary Ban**
+
+   Cause: A serious violation of community standards, including sustained inappropriate behavior.
```

Comment:
I think the Causes should be in the same general format so people can see the escalation. So change to "A violation through a single serious incident or sustained inappropriate behavior".